.github/workflows/pr-title-checker.yml

```diff
@@ -0,0 +1,19 @@
+name: "PR Title Checker"
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - labeled
+      - unlabeled
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: webex/pr-title-checker
```

Comment:
https://github.com/webex/pr-title-checker
Forked in Webex and checking if workflow is trigerring or not.